doc_user/UserManual.adoc

```diff
@@ -828,6 +829,409 @@ image:image251.png[success]
+
+image:image255d.png[Define a lot]
+
+The procedure is pretty much the same as defined in chapter <<insert-stock-charging-movement,4.2.2.2 Insert Stock Charging Movement>>.
+
+Once the lot is defined, a real quantity greater than zero must be specified, otherwise validation will fail:
+
+image:image255e.png[Validation fail]
+
+===== 4.4.2.2 Edit Inventory ([.underline]##E##dit Inventory)
+
+To modify an inventory, select the inventory in the *_Inventory Browser_* window. Once the inventory has been highlighted, click on *[.underline]##E##dit* button to enter on the *_Edit Inventory_* window. Now the record is available for changes. This function allows editing all the data of the inventory including inventory(ies) row(s) of the inventory.
+
+NOTE: only inventories in DRAFT and VALIDATED states can be edited.
+
+image:image256.png[Edit Inventory]
+
+===== 4.4.2.3 Validate Inventory ([.underline]##V##alidate)
+
+During modification it is possible to validate the inventory by clicking on the *[.underline]##V##alidate* button. This may result in the following dialog:
+
+image:image271.png[Error]
+
+If the inventory has unsaved changes.
+
+image:image266.png[Validate Inventory]
+
+This Information window asks whether to check for any medicals on the stock or just the ones that are in the list?
+
+- *Yes, all medicals*: the system will check if new movements are made on the main store for any medical and eventually show the changes in a new dialog. See below.
+- *No, only the medicals in the list*: the system will check if new movement are made on the main store for the specific medicals that are in the list. E.g. see the image below. 
+
+image:image272.png[Validate]
+
+Click on the *[.underline]##O##k* button to continue. The system may also show another information message that specifies the date of the inventory will be updated as a new more recent movement has been found:
+
+image:image273.png[Inventory date update]
+
+NOTE: Since the inventory will result in a number of charges/discharges in <<pharmaceutical-stock,Pharmaceutical Stock>> to align the stock levels to the real quantities (physical count), its date cannot be prior an already existing movement, so the system will always check if the current inventory is the latest operation to be recorded.
+
+Click on the *[.underline]##O##k* button to continue. If new movements have been found, the system will show the following dialog in order to update the current inventory:
+
+image:image274.png[Question]
+
+By clicking the *[.underline]##Y##es* button the system will actualize the inventory by adding new inventory row(s) and updating its date/time and status, see the picture below:
+
+image:image275.png[validate Inventory]
+
+Otherwise, by clicking the *[.underline]##N##o* button, nothing will be changed, but the inventory will remain (or go back) to DRAFT state.
```

Comment:
```suggestion
Otherwise, by clicking the *[.underline]##N##o* button, nothing is changed and the inventory remains (or goes back) to DRAFT state.
```